benchmarks/src/tpch/convert.rs

```diff
@@ -88,9 +88,8 @@ impl ConvertOpt {
                 .schema()
                 .iter()
                 .take(schema.fields.len() - 1)
-                .map(|(qualifier, field)| {
-                    Expr::Column(Column::from((qualifier, field.as_ref())))
-                })
+                .map(Column::from)
```

Comment:
can we still use a single map?

Reply:
Good call -- I added some more `Expr::from` impls in  c4bc7e36d and now this looks pretty nice